packages/delivery-utils/src/array-from-node-list.test.ts

```diff
@@ -0,0 +1,38 @@
+import { arrayFromNodeList } from './array-from-node-list';
+
+describe('Array From NodeList Test', () => {
+    document.body.innerHTML = `
+    <ul>
+        <li>Test 0</li>
+        <li>Test 1</li>
+        <li>Test 2</li>
+        <li>Test 3</li>
+    </ul>
+    `;
```

Comment:
In general you want to wrap set up logic like this in a `beforeEach()` so that it is reset for each test. But especially so since you are mutating the value of DOM nodes below on line 30.